src/stats/probs/cdf.rs

```diff
@@ -199,6 +206,12 @@ impl<T: Ord> CDF<T> {
 
     /// Downsample CDF to n entries. Panics if n <= 1 and returns identity if n is greater
     /// than the number of entries.
+    ///
+    /// Runtime complexity: O(m), where m is the original number of `Entry`s in `CDF`.
+    ///
+    /// # Arguments
+    ///
+    /// * `n` - Number of entries after downsampling.
```

Comment:
come to think of it, it's not as trivial as it seems, since because of step_by actually only n elements are looked at, however, I think step_by works by calling iterator.next() repeatedly and discarding the appropriate amount of entries yielded. On the other hand, the compiler might optimize this away.

Reply:
Well, then I guess `O(m)` is a rather conservative bound, for now...